### PR TITLE
New applet BgMedia@spacy01

### PR DESCRIPTION
**Pull request by spacy01**
This is applet for streaming bulgarian radios and TVs.

### CODE REVIEW

**Comment by jaszhix** [comment]
The [folder structure is incorrect](https://github.com/linuxmint/cinnamon-spices-applets#file-structure), applet files belong in `<uuid>/files/<uuid>`. 

What differences are there between this and BgRadio@spacy01? I didn't see a readme file.

File: BgMedia@spacy01/applet.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+const St = imports.gi.St;
+const Cinnamon = imports.gi.Cinnamon;
+
+Gettext.bindtextdomain(UUID, GLib.get_home_dir() + "/.local/share/locale")
+
+function _(str) {
+  return Gettext.dgettext(UUID, str);
+}
+
+function _(text) {
```
Comment (jaszhix) [inline BgMedia@spacy01/applet.js]: `_` is being declared twice.

**Comment by NikoKrause** [comment]
I also have concerns about the many bash scripts.
Can't you do this in the applet itself. Or squash in one script with arguments?

Also checkout the validate-spices script mentioned in the README of this repo.

**Comment by spacy01** [comment]
@jaszhix, @NikoKrause,  thank you for your recommendations. 

I'm not a really coder, you may be figured out already. And GitHub is still a scary place for me. Don't be too harsh, please. :)

I'll update my old applet instead this one. I also will try to get rid of all bash scripts. But i faced with problem. And need some help.

For all my tv streams (exept one) i need for rtmpdump. So, when rtmpdump is in the bash script, all is working, but when is in the applet, like this:

`Main.Util.spawnCommandLine("rtmpdump -r 'http://ios.cdn.bg:2020/fls' -a 'fls' -f 'WIN 11,5,502,149' -p 'http://cdn.bg/live/8W5GA2ujvB' -W 'http://cdn.bg/eflash/jwplayer510/player.swf' -y 'bnt.stream?at=b0d1270b39e08ad9c78dc53f43a1ba5c' -b '0' | vlc -");`

Not working. The applet is load with no errors, but tv stream not start. Any ideas? Thank you!

**Comment by jaszhix** [comment]
Without having having tested this, try wrapping the command with `"bash -c \"rtmpdump -r 'http://ios.cdn.bg:2020/fls' -a 'fls' -f 'WIN 11,5,502,149' -p 'http://cdn.bg/live/8W5GA2ujvB' -W 'http://cdn.bg/eflash/jwplayer510/player.swf' -y 'bnt.stream?at=b0d1270b39e08ad9c78dc53f43a1ba5c' -b '0' | vlc -\""`. Inner quotes need to be escaped.

**Comment by spacy01** [comment]
@jaszhix Thank you! It's working!

**Pull request closed without merging**